README.md

```diff
@@ -566,6 +566,8 @@ An up-to-date list of the convenience operations can be found [here](https://doc
 #### NGS10 (context information availability)
 | Method | Path                                                                                     | Action |
 | ------ |:--------------------------------------------------------------------------------------- | ---:|
+| GET    | /v1/contextentities                                                                  | R |
```

Comment:
I understand that PEP is not case sensitive but maybe for coherence in this table you should use the usual style in CB operations. i.e.:

```
/v1/contextEntities
```

BTW, I'm almost sure that Orion is case sensitive in operations ( @kzangeli could confirm definetively)